sdk/inc/azure/core/_az_cfg.h

```diff
@@ -2,9 +2,10 @@
 // SPDX-License-Identifier: MIT
 
 /**
- * @file _az_cfg.h
+ * @file
```

Comment:
I think it is better to have it, because otherwise, if the first entity in the header is not documented, it is for sure will be associated with that entity. When we have a `@file`, we clearly mark that the comment section should be applied to current file description.

Reply:
> because otherwise, if the first entity in the header is not documented, it is for sure will be associated with that entity.

Can you clarify what happens if the first entity in the header is **not** documented and we **don't** have `@file`? How do we benefit from the association or how does the user/generated docs benefit?

Also, once we turn on docs validation in CI, we wouldn't have missing docs anyway (or else CI would fail).